src/test/unit/parsers/test_event_parser.py

```diff
@@ -264,3 +264,143 @@ def test_parse_event_api_gw_v2():
+                    "SigningCertUrl": "https://sns.us-east-2.amazonaws.com/SimpleNotificationService-ac565b8b1a6c5d002d285f9598aa1d9b.pem",
+                    "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
+                    "Message": "Hello from SNS1!",
+                    "MessageAttributes": {
+                        "Test": {"Type": "String", "Value": "TestString"},
+                        "TestBinary": {"Type": "Binary", "Value": "TestBinary"},
+                    },
+                    "Type": "Notification",
+                    "UnsubscribeUrl": "https://sns.us-east-2.amazonaws.com/?Action=Unsubscribe&amp;SubscriptionArn=arn:aws:sns:us-east-2:123456789012:test-lambda:21be56ed-a058-49f5-8c98-aedd2564c486",
+                    "TopicArn": "arn:aws:sns:us-east-2:123456789012:sns-lambda",
+                    "Subject": "TestInvoke",
+                },
+            },
+            {
+                "EventVersion": "1.0",
+                "EventSubscriptionArn": "arn:aws:sns:us-east-2:123456789012:sns-lambda:21be56ed-a058-49f5-8c98-aedd2564c486",
+                "EventSource": "aws:sns",
+                "Sns": {
+                    "SignatureVersion": "1",
+                    "Timestamp": "2019-01-02T12:45:07.000Z",
+                    "Signature": "tcc6faL2yUC6dgZdmrwh1Y4cGa/ebXEkAi6RibDsvpi+tE/1+82j...65r==",
+                    "SigningCertUrl": "https://sns.us-east-2.amazonaws.com/SimpleNotificationService-ac565b8b1a6c5d002d285f9598aa1d9b.pem",
+                    "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
+                    "Message": "Hello from SNS2!",
+                    "MessageAttributes": {
+                        "Test": {"Type": "String", "Value": "TestString"},
+                        "TestBinary": {"Type": "Binary", "Value": "TestBinary"},
+                    },
+                    "Type": "Notification",
+                    "UnsubscribeUrl": "https://sns.us-east-2.amazonaws.com/?Action=Unsubscribe&amp;SubscriptionArn=arn:aws:sns:us-east-2:123456789012:test-lambda:21be56ed-a058-49f5-8c98-aedd2564c486",
+                    "TopicArn": "arn:aws:sns:us-east-2:123456789012:sns-lambda",
+                    "Subject": "TestInvoke",
+                },
+            },
+        ]
+    }
+
+    order_sns_event = EventParser.parse_event(event=not_order_sns_event)
+
+    assert json.dumps(order_sns_event) == json.dumps(
+        OrderedDict(
+            {
+                "Records": [
+                    {
+                        "Sns": {
+                            "Message": "Hello from SNS1!",
+                            "MessageAttributes": {
+                                "Test": {"Type": "String", "Value": "TestString"},
+                                "TestBinary": {"Type": "Binary", "Value": "TestBinary"},
+                            },
+                            "MessageId": "95df01b4-ee98-5cb9-9903-4c221d41eb5e",
```

Comment:
I think that we prefer to add the `timestamp` too, but it's not mandatory...

Reply:
Efi prefer not to add